### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by rapp-fzi**
created initial workflow file.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  Metamodel-Modeling-Foundations:
+    uses:  MDSD-Tools/Palladio-Build-ActionsPipeline/.github/workflows/build.yml@v3.2
+    with:
+      use-display-output: true
+      no-caching: true
+      java-version: 17
+      runner-label: ubuntu-latest
+      deploy-updatesite: 'releng/tools.mdsd.probdist.updatesite/target/repository'
```
Comment (Nicolas-Boltz) [inline .github/workflows/main.yml]: This is the wrong path. If you have a look in the old jenkinsfile you can see it should be:
'releng/tools.mdsd.modelingfoundations.updatesite/target/repository'

**Review by Nicolas-Boltz — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Build Pipeline
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  Metamodel-Modeling-Foundations:
+    uses:  MDSD-Tools/Palladio-Build-ActionsPipeline/.github/workflows/build.yml@v3.2
```
Comment (Nicolas-Boltz) [inline .github/workflows/main.yml]: Also, there is noch Palladio-Build-ActionsPipeline repository in MDSD-Tools. Here you should use the pipeline that is also used in the Palladio organization.
Just change MDSD-Tools to PalladioSimulator

**Review by Nicolas-Boltz — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (rapp-fzi) [reply .github/workflows/main.yml]: done

File: .github/workflows/main.yml (hunk 1 above)
Comment (rapp-fzi) [reply .github/workflows/main.yml]: done

**Review by Nicolas-Boltz — APPROVED** [review]
LGTM

**Pull request merged**